### PR TITLE
fix: ignore error after close & register channel issue

### PR DESCRIPTION
**Pull request by gxcsoccer**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

1. 忽略主动关闭后的异常（主要是在单测场景下）
2. 增加最大空闲时间
3. 修复 Follower 到 Leader 注册通道的 bug
4. Leader close 后不 removeAllListeners

### CODE REVIEW

File: lib/follower.js [hunk 1]
```diff
@@ -95,6 +98,14 @@ class Follower extends Base {
     };
   }
 
+  send(...args) {
+    // 主动关闭以后不再执行 send
```
Comment (dead-horse) [inline lib/follower.js]: 英文？

Comment (gxcsoccer) [inline lib/follower.js]: 额，我改掉

**Comment by popomore** [comment]
还没合

**Comment by gxcsoccer** [comment]
> Windows ci 挂了是否要忽略？

先忽略

**Comment by gxcsoccer** [comment]
\+ cluster-client@1.6.5

**Pull request merged**